### PR TITLE
Simplified Nutrition blacklists calcium tablets and multivitamins

### PR DESCRIPTION
**Pull request by ghost**
Calcium tablets and multivitamins only have a use with vitamins enabled, so I wanted to scrap these for the novitamins mod because they have no purpose and spawn frequently.

Gummy vitamins still spawn, due to having a fun value.

### CODE REVIEW

File: data/mods/novitamins/modinfo.json [hunk 1]
```diff
@@ -11,5 +11,9 @@
   {
     "type": "WORLD_OPTION",
     "options": [ "NO_VITAMINS" ]
+  },
+  {
+    "type": "ITEM_BLACKLIST",
+    "items": [ "calcium_tablet", "vitamins" ], "//": "Gummy vitamins not included here due to having an enjoyment level"
```
Comment (ZhilkinSerg) [inline data/mods/novitamins/modinfo.json]: You would like to lint file here - http://dev.narc.ro/cataclysm/format.html.

**Comment by pisskop** [comment]
Multivitamins and gummy vitamins should both raise health independent of their nutritional value.

**Comment by smolbird** [comment]
Original use of vitamins prior to the nutrition system was a health boost mechanic, this should be retained.

**Comment by Xhuis** [comment]
I agree about it being good to have a health increase; I'll do that.

File: data/mods/novitamins/modinfo.json (hunk 1 above)
Comment (Xhuis) [reply data/mods/novitamins/modinfo.json]: I should write the website's URL on some paper and put it in my pocket. Then I could have pocket lint!

**Comment by smolbird** [comment]
It used to have it hardcoded in the use action actually.

**Comment by Xhuis** [comment]
I did some testing, and having it in the json *does* appear to still work. It seems as though it'd be cleaner that way.

**Comment by Cyrano7** [comment]
@Xhuis my understanding of why "healthy" was removed in the first place was because since you could eat tons of vitamins without getting full you could essentially super boost your health by eating fistfuls of vitamins. You could make the vitamin "healthy" value a item override in the mod itself actually which might be a better fit. That's the part of this PR that's probably holding it up from being merged. The part about blacklisting calcium tablets in the mod i'm sure everyone agrees with.

**Comment by Xhuis** [comment]
That sounds fair. I'm visiting family until Saturday, though, so it'll take me a few days to get that changed unless I get some time to do it with webeditor or something.

**Comment by Cyrano7** [comment]
@Xhuis If you want my 2 cents i'd say just blacklist both the multivitamin and calcium tablet for the mod.

**Comment by Xhuis** [comment]
This should be ready now, unless there's any other issues with it?

**Comment by Xhuis** [comment]
Thank you!

**Pull request merged**